src/components/#USBControllerButton.tsx#

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react'
```

Comment:
Why does this file name have `#` prefix and suffix?

Reply:
ummm, stupid backup file, I need to delete it.